src/calculateRentalCost.js

```diff
@@ -3,8 +3,19 @@
  *
  * @return {number}
  */
+const discountsDays1 = 3;
+const discountsDays2 = 7;
+const discount1 = 20;
+const discount2 = 50;
+
 function calculateRentalCost(days) {
-  // write code here
+  if (days < discountsDays1) {
+    return days * 40;
+  } else if (days >= discountsDays1 && days < discountsDays2) {
+    return days * 40 - discount1;
+  } else if (days >= discountsDays2) {
+    return days * 40 - discount2;
```

Comment:
The magic number `40` representing the daily rental cost should be replaced with a constant to make the code more readable and maintainable. Define a constant like `DAILY_RENTAL_RATE` at the beginning of your code and use it in place of the literal value.